proto/stream.proto

```diff
@@ -1,17 +1,23 @@
-syntax = "proto2";
+syntax = "proto3";
 
 package pb;
 
-import "metadata.proto";
-import "source.proto";
-
+import "fee.proto";
+import "file.proto";
 
 message Stream {
-    enum Version {
-        UNKNOWN_VERSION = 0;
-        _0_0_1 = 1;
-    }
-    required Version version = 1;
-    required Metadata metadata = 2;
-    required Source source = 3;
+    bytes hash = 1;
+    string language = 2;
+    string title = 3;
+    string author = 4;
+    string description = 5;
+    string media_type = 6;
+    string license = 7;
+    File file = 8;
+
+    Fee fee = 16;
+    string license_url = 17;
+    string thumbnail_url = 18;
+    uint32 duration = 19;
```

Comment:
content height/length and orientation would be helpful for spee.ch and other apps too - see https://github.com/lbryio/spee.ch/issues/435 / https://github.com/lbryio/spee.ch/issues/527. We wouldn't have to run ffmpeg on new files anymore...

Reply:
per @lyoshenka, we should add these under a dimensions subfield

Reply:
what are the possible values for `orientation`?

Reply:
@lyoshenka I would  say horizontal/vertical, but we can expand that to 0/90/180/270 (https://stackoverflow.com/a/47676129/8308000). I asked Sean and he suggested a boolean for a Portrait setting.